ui/app/AppLayouts/Wallet/views/LeftTabView.qml

```diff
@@ -333,37 +335,50 @@ Rectangle {
                             text: qsTr("All accounts")
                             font.weight: Font.Medium
                             font.pixelSize: 15
+                            lineHeightMode: Text.FixedHeight
+                            lineHeight: 22
                         }
-
-                        StatusTextWithLoadingState {
-                            id: walletAmountValue
-                            objectName: "walletLeftListAmountValue"
-                            customColor: Style.current.textColor
-                            text: LocaleUtils.currencyAmountToLocaleString(RootStore.totalCurrencyBalance)
-                            font.pixelSize: 22
-                            loading: RootStore.assetsLoading
+                        RowLayout {
+                            spacing: 4
+                            StatusTextWithLoadingState {
+                                id: walletAmountValue
+                                objectName: "walletLeftListAmountValue"
+                                customColor: Style.current.textColor
+                                text: LocaleUtils.currencyAmountToLocaleString(RootStore.totalCurrencyBalance, {noSymbol: true})
+                                font.pixelSize: 22
+                                loading: RootStore.assetsLoading
+                                font.weight: Font.Medium
+                                lineHeightMode: Text.FixedHeight
+                                lineHeight: 38
+                                verticalAlignment: Text.AlignVCenter
+                            }
+                            StatusTextWithLoadingState {
+                                customColor: Style.current.textColor
+                                text: RootStore.totalCurrencyBalance.symbol
+                                font.pixelSize: 13
+                                loading: RootStore.assetsLoading
+                                font.weight: Font.Medium
+                                lineHeightMode: Text.FixedHeight
+                                lineHeight: 38
+                                verticalAlignment: Text.AlignBottom
+                                bottomPadding: Style.current.padding
+                            }
                             visible: !networkConnectionStore.accountBalanceNotAvailable
-                            anchors.top: allAccounts.bottom
-                            anchors.topMargin: 4
-                            anchors.bottomMargin: Style.current.padding
-                            leftPadding: Style.current.padding
                         }
-
                         StatusFlatRoundButton {
                             id: errorIcon
-                            width: 14
-                            height: visible ? 14 : 0
+                            Layout.preferredWidth: 14
+                            Layout.preferredHeight: visible ? 14 : 0
```

Comment:
The `visible` condition shouldn't be needed; if you're inside a `RowLayout` and `visible` is `false`, then you'll never see this button